src/input/InputManager.js

```diff
@@ -971,8 +971,9 @@ var InputManager = new Class({
         y += gameObject.displayOriginY;
 
         var input = gameObject.input;
+        var hitAreaCallback = input?.hitAreaCallback;
```

Comment:
Unfortunately, **Optional chaining operator (?.)** is not **ES5**-compliant, so it can't be used in Phaser 3 code.

Reply:
got it noted and updated